### PR TITLE
Update logging-config.asciidoc make it possible to directly remove deprecations logs without further googling.

### PR DESCRIPTION
**Pull request by nicholasklem**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by rjernst** [comment]
@pgomulka Can you please review this?

**Comment by pgomulka** [comment]
@elasticmachine ok to test

File: docs/reference/setup/logging-config.asciidoc [hunk 1]
```diff
@@ -212,7 +212,14 @@ logs to roll and compress after 1 GB, and to preserve a maximum of five log
 files (four rolled logs, and the active log).
 
 You can disable it in the `config/log4j2.properties` file by setting the deprecation
-log level to `error`.
+log level to `error` like this:
+```
```
Comment (pgomulka) [inline docs/reference/setup/logging-config.asciidoc]:
```suggestion
[source,properties]
--------------------------------------------------
```

File: docs/reference/setup/logging-config.asciidoc [hunk 2]
```diff
@@ -212,7 +212,14 @@ logs to roll and compress after 1 GB, and to preserve a maximum of five log
...
+...
+logger.deprecation.name = org.elasticsearch.deprecation
+logger.deprecation.level = error
+...
+```
```
Comment (pgomulka) [inline docs/reference/setup/logging-config.asciidoc]:
```suggestion
--------------------------------------------------
```

File: docs/reference/setup/logging-config.asciidoc [hunk 3]
```diff
@@ -212,7 +212,14 @@ logs to roll and compress after 1 GB, and to preserve a maximum of five log
 files (four rolled logs, and the active log).
 
 You can disable it in the `config/log4j2.properties` file by setting the deprecation
-log level to `error`.
+log level to `error` like this:
+```
+...
```
Comment (pgomulka) [inline docs/reference/setup/logging-config.asciidoc]: I guess the context is clear that this is part of a bigger config. How about we remove the ... ?

**Review by pgomulka — CHANGES_REQUESTED** [review]
it looks good! thank you for improving. I left few comments

**Comment by pgomulka** [comment]
ok to test

**Comment by pgomulka** [comment]
ok to test

**Comment by pgomulka** [comment]
ok to test

**Comment by pgomulka** [comment]
@elasticmachine run elasticsearch-ci/1

**Comment by pgomulka** [comment]
@elasticmachine run elasticsearch-ci/1

**Comment by pgomulka** [comment]
@nicholasklem once again - thank you for updating this. I have merged your change and will update the corresponding branches in older versions

**Pull request merged**